pkg/schedule/schedulers/hot_region_test.go

```diff
@@ -724,14 +723,17 @@ func TestHotWriteRegionScheduleByteRateOnlyWithTiFlash(t *testing.T) {
 			// balance by leader selected
 			re.Equal("transfer-hot-write-leader", op.Desc())
 			operatorutil.CheckTransferLeaderFrom(re, op, operator.OpHotRegion, 1)
+			hasLeaderOperator = true
 		case 2:
 			// balance by peer selected
-			re.Equal("move-hot-write-leader", op.Desc())
-			operatorutil.CheckTransferLearner(re, op, operator.OpHotRegion, 8, 10)
+			re.Equal("move-hot-write-peer", op.Desc())
+			operatorutil.CheckTransferLearner(re, op, operator.OpHotRegion, 8, 0)
```

Comment:
it may be scheduled to store 10 or store 11

Reply:
Is it possible to make the result deterministic？

Reply:
done